performance-testing/performance-testing-aws/src/main/java/uk/gov/gchq/gaffer/performancetesting/aws/CloudWatchMetricsListener.java

```diff
@@ -112,6 +112,7 @@ public void update(final Metrics metrics) {
     @Override
     public void close() {
         this.cloudwatch.shutdown();
```

Comment:
Is it possible for close to be called twice? If so we would need a null check around ```this.cloudwatch.shutdown();```

Similarly in a 'disconnect' method below.